tensorflow/python/kernel_tests/scatter_nd_ops_test.py

```diff
@@ -96,6 +96,14 @@ def _NumpyDiv(ref, indices, updates):
   return _NumpyScatterNd(ref, indices, updates, lambda p, u: p / u)
 
 
+def _NumpyMin(ref, indices, updates):
+  return _NumpyScatterNd(ref, indices, updates, lambda p, u: min(p, u))
```

Comment:
There is no need for lambda here, `lambda p, u: min(p, u)` is exactly the same as `min`.

Reply:
I've changed the call and removed the lambda function.